pubsub/google/cloud/pubsub_v1/publisher/_sequencer/unordered_sequencer.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2019, Google LLC All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud.pubsub_v1.publisher._sequencer import base
+
+
+class UnorderedSequencer(base.Sequencer):
+    """ Sequences messages into batches for one topic without any ordering.
+
+        Public methods are NOT thread-safe.
```

Comment:
Bump.

Reply:
Ordered sequencers need thread-safety because there's a commit thread in thread.Batch that calls back into it.

Unordered sequencers call batch.commit() and forget about it, so they don't need thread-safety. I mainly omitted locks here to reduce locking costs.